src/coreclr/jit/CMakeLists.txt

```diff
@@ -626,7 +626,14 @@ endif()
 if (CLR_CMAKE_TARGET_WIN32 AND CLR_CMAKE_PGO_INSTRUMENT)
   # Copy PGO dependency to target dir
   set(PGORT_DLL "pgort140.dll")
-  find_path(PGORT_DIR ${PGORT_DLL} REQUIRED)
+  cmake_path(SET PGORT_DIR_ARM64 "$ENV{VCToolsInstallDir}bin/arm64")
+  if(CLR_CMAKE_TARGET_ARCH_ARM64)
+    cmake_path(SET PGORT_DIR "$ENV{VCToolsInstallDir}bin/arm64")
+  elseif(CLR_CMAKE_TARGET_ARCH_I386)
+    cmake_path(SET PGORT_DIR "$ENV{VCToolsInstallDir}bin/Hostx86/x86")
```

Comment:
This should probably work, but I think we only support building on x64, so technically this should be `Hostx64/x86`

Reply:
I will fix it up. Want to do it right :)

Reply:
Actually, `Hostx64/x86` does not contain `pgort140.dll`. So I am going to stick with what I have.